core/trino-parser/src/main/java/io/trino/sql/tree/DropColumn.java

```diff
@@ -26,21 +26,21 @@
         extends Statement
 {
     private final QualifiedName table;
-    private final Identifier column;
+    private final QualifiedName column;
```

Comment:
It's a "column or field" now.
Let's rename to `field`.
(a column is a field in a table; a row field is not a column; the "field" term is more generic)